<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>633</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Academy Engraved LET</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 100, 100);
  border: 1px solid black
;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="title">
     <string>Funcs</string>
    </property>
    <widget class="QPushButton" name="pushButton_cos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  background-color: rgb(215, 100, 100);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
     </property>
     <property name="text">
      <string>cos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_log">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tan">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>tan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>sin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_atan">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>atan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_asin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>asin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sqrt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>sqrt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_acos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>acos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ln">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>ln</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_credit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>credit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_depos">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>depos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_GRAPH">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(200, 100, 100);
	color: white;
	border: 2px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
</string>
    </property>
    <property name="title">
     <string>Arifmetic</string>
    </property>
    <widget class="QPushButton" name="pushButton_Bracket_Left">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mul">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_min">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Bracket_Right">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_dev">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mod">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>mod</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sum">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_min_or_plas">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_dot">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rm">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_x">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>271</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="title">
     <string>Numbers</string>
    </property>
    <widget class="QPushButton" name="pushButton_AC">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_equally">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient ( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="result_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>671</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
